### ISSUE TITLE
Can't get into scroll mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [breanadoll/iphodroid#20]
```
Still doesn't work with pwnage tool. My results from hours of testing: iPhone 
2G doesn't work jailbroken with redsn0w, backra1n, or PWnage tool using v.0.5 
R1 - v.0.6 R10. Please fix this bug, with this bug nothing works.

My specs:
iPhone 2G with version 3.1.2
Jailbreak: blackra1n, redsn0w, PWnage tool
Computer: PPC iMac G5 10.5.8  Leopard
Version: 0.5-0.6 R10
Problem: Everything works until I get to the Openiboot step where the iPhone 
just goes into connect to itunes or a picture of Steve Jobs on PWnage tool. 
Basically I can't get into scroll mode.
Solution: Don't try (I.E. doesn't have a solution yet)

Please fix this.
```

Original issue reported on code.google.com by `GTGi...@gmail.com` on 20 Jun 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
Same here uploads all the files then when i get to the scroll mode it just 
boots and opens itunes saying a phone is in recovery!!!   Is a fix available?
```

Original comment by `KEITH.LA...@gmail.com` on 21 Jun 2010 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
Im about to update a new release today R11. Follow me 
www.twitter.com/sergiomcfly
```

Original comment by `sergiomcfly` on 21 Jun 2010 at 6:49
